plugin/k8s_external/README.md

```diff
@@ -106,6 +113,16 @@ zone transfers.  Notifies are not supported.
      }
  ~~~
 
+With the fallthrough option, if the query is not found, it will be passed to the next plugin.
+~~~
+. {
+   kubernetes cluster.local
+   k8s_external example.org {
+     fallthrough
+   }
```

Comment:
Should include a next plugin in this example (e.g. _forward_), otherwise the example only accomplishes turning NXDOMAIN responses into SERVFAIL responses.